src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -26,6 +26,17 @@
  */
 final class ProgressBar
 {
+    public const VERBOSE = 'verbose';
+    public const VERY_VERBOSE = 'very_verbose';
+    public const DEBUG = 'debug';
+    public const NORMAL = 'normal';
+
+    const VERBOSE_NOMAX = 'verbose_nomax';
+    const VERY_VERBOSE_NOMAX = 'very_verbose_nomax';
+    const DEBUG_NOMAX = 'debug_nomax';
+    const NORMAL_NOMAX = 'normal_nomax';
+    const NOMAX_SUFIX = '_nomax';
```

Comment:
Without an explicit visibility modifier, the constants are still public. 🙃 

`private` is the modifier you're looking for.

Reply:
🤦‍♂️
Will fix